extension/chrome/settings/modules/backup-manual-module.ts

```diff
@@ -53,17 +54,78 @@ export class BackupManualActionModule extends ViewModule<BackupView> {
     }
   }
 
+  private addKeyToBackup = (prvKeyIdentity: PrvKeyIdentity) => {
+    this.view.prvKeysToManuallyBackup.push(prvKeyIdentity);
+  }
+
+  private removeKeyToBackup = (fingerprints: string[]) => {
+    this.view.prvKeysToManuallyBackup.splice(this.view.prvKeysToManuallyBackup.findIndex(prvIdentity => prvIdentity.fingerprints === fingerprints), 1);
+  }
+
+  private preparePrvKeysBackupSelection = async () => {
+    const primaryKeys = await KeyStore.getAllWithOptionalPassPhrase(this.view.acctEmail);
+    if (primaryKeys.length > 1) {
+      await this.renderPrvKeysBackupSelection(primaryKeys);
+    } else {
+      this.addKeyToBackup({ 'email': String(primaryKeys[0].emails), 'fingerprints': primaryKeys[0].fingerprints });
+    }
+  }
+
+  private renderPrvKeysBackupSelection = async (primaryKeys: ExtendedKeyInfo[]) => {
+    for (const primaryKi of primaryKeys) {
+      const email = Xss.escape(String(primaryKi.emails));
+      const fingerprints = primaryKi.fingerprints;
+      const keyType =  (await KeyUtil.parse(primaryKi.private)).type;
+      const dom = `
+      <div class="mb-20">
+        <div class="details">
+          <label>
+            <p class="m-0">
+            <input class="input_prvkey_backup_checkbox" type="checkbox" checked data-emails="${email}" data-fingerprints="${fingerprints}" ${keyType === 'x509' ? 'disabled' : ''} />
+            ${email}
+            </p>
+            <p class="m-0 prv_fingerprint"><span>${keyType} - ${email} - ${Str.spaced(fingerprints[0])}</span></p>
+          </label>
+        </div>
+      </div>
+      `.trim();
+      $('.key_backup_selection').append(dom); // xss-escaped
+      if (keyType !== 'x509') {
+        this.addKeyToBackup({ 'email': email, 'fingerprints': fingerprints });
```

Comment:
no need to quote property names. In this case `{ email, fingerprints }` should do, I think.

Reply:
It works! Thank you!